dev-docs/source/howto/install.rst

```diff
@@ -0,0 +1,127 @@
+Installation
```

Comment:
By default the highlighting is for the Python domain. In this file, however, all code blocks are for the console. Hence start this file with:

```
.. highlight:: console
```

To properly highlight the code blocks

Reply:
Oh, don't use `bash`. That's for scripts. Interaction on the console should use `console`

Reply:
thanks, didn't know about "console" as language domain. Fixed it everywhere!